src/Utility/StringUtil.ts

```diff
@@ -60,4 +63,22 @@ export class StringUtil {
 
     return buffer;
   }
+
+  public static validateDBName(dbName?: string): void {
+    if (TypeUtil.isNone(dbName) || !dbName) {
+      throw new InvalidOperationException('Empty name is not valid');
+    }
+
+    if (!/^[A-Za-z0-9_\-\.]+$/.test(dbName)) {
+      throw new InvalidOperationException('Database name can only contain only A-Z, a-z, \"_\", \".\" or \"-\"');
+    }
+  }
+
+  public static ucFirst(string: string): string {
```

Comment:
Use full method names. Without reading the code, there is no way to translate `uc` to upperCase`